source/sass/blocks/form.scss

```
@@ -1,19 +1,44 @@
 .form {
   overflow: hidden;
+  padding-bottom: 12px;
+
+  @media (min-width: $tablet-width) {
+    padding-bottom: 0;
+  }
+}
+
+.form__page {
+  @media (min-width: $tablet-width) {
+    padding-left: 70px;
+    padding-right: 67px;
+  }
+
+  @media (min-width: $desktop-width) {
+    padding-left: 40px;
+    padding-right: 40px;
+  }
 }
 
 .form__description {
   font-weight: 400;
   font-size: 14px;
   line-height: 18px;
-  padding-top: 38px;
-  padding-bottom: 39px;
+  padding-top: 0;
+  padding-bottom: 0;
   text-transform: uppercase;
 
   @media (min-width: $tablet-width) {
     font-size: 20px;
     line-height: 30px;
-    padding-top: 20px;
+    margin-left: 0;
+    margin-right: 0;
+    margin-bottom: 10px;
+    max-width: 500px;
+    padding-left: 70px;
+  }
+
+  @media (min-width: $desktop-width) {
+    padding-left: 111px;
```

Тут обнуление отступов и паддинги можно удалить: работу делает как раз центровщик.
Останется только отступ снизу, а медиа-запрос для десктопа не нужен

```suggestion
    margin-bottom: 10px;
```